testing/gradle/gradle

```diff
@@ -0,0 +1,7 @@
+#!/bin/sh
+APP_NAME="Gradle"
+APP_BASE_NAME=`basename "$0"`
```

Comment:
Sorry for that one - this was derived from Gradle's own /usr/bin script, just trimmed down heavily as we don't need things like Darwin compatibility.